### PR TITLE
Add the Svelte tutorial

### PR DESCRIPTION
**Pull request by metonym**
This draft PR adds the Svelte tutorial under the "Developing" menu in the Carbon website.

The source code for the tutorial may be found here: https://github.com/carbon-design-system/carbon-tutorial-svelte

Netlify preview for the completed tutorial: https://carbon-tutorial-svelte.netlify.app/

---

**Todo**

- [x] Overview
- [x] Step 1
- [x] Step 2
- [x] Step 3
- [x] Step 4
- [x] Step 5
- [x] Wrapping up

**carbon-tutorial-svelte**

- [x] Add `design.sketch` file to source code for steps 2-6
- [x] Add missing classes to third row in landing page for steps 3, 4
- [x] Use dedicated `svelte-step-X--carbon-tutorial-svelte.netlify.app/` for previews
- [x] Set up mergify to auto-approve + label passing PRs for the badging process
- [x] Set up CircleCI for steps 1-5

**Svelte tutorial badge**

- [x] Update "Wrapping up" with the Svelte badge
- [ ] Update "Wrapping up" with the Acclaim link
- [ ] Update https://badges.carbondesignsystem.com/ to include Svelte badge

**Peer Review Feedback**

- [ ] Directly import Carbon icons

### CODE REVIEW

**Comment by mattrosno** [comment]
@jeanservaas can you make the website image and the Acclaim badge image for the Svelte tutorial? https://www.carbondesignsystem.com/developing/react-tutorial/wrapping-up/#application-form

**Comment by mattrosno** [comment]
@jeanservaas can you create the "Svelte" images for 1) website, and 2) badge application?

https://www.carbondesignsystem.com/developing/react-tutorial/wrapping-up/#application-form

https://www.credly.com/org/ibm/badge/ibm-carbon-design-system-developer-essentials-angular

Then I can put in the badge application form with Credly (previously Acclaim).

**Comment by jeanservaas** [comment]
DE_svelte

**Comment by metonym** [comment]
@jeanservaas Thank you – I've added the artwork to the site.

[Latest deployment](https://carbondesignsystem-guvmmelid-carbon-design-system.vercel.app/developing/svelte-tutorial/wrapping-up#application-form)

**Comment by tay1orjones** [comment]
@metonym @mattrosno @jeanservaas this one's quite old but not totally stale. Is there still interest to try and get this in? 

We recently removed the badging from all the tutorials and it looks like that was the only remaining piece for this based on the task/check list.

**Comment by mattrosno** [comment]
If the content is good and the tutorial can be done without errors, they it could be worth getting it merged in without the badging. Can we have somebody verify the content prior to merging it in or closing this PR?

**Comment by metonym** [comment]
@tay1orjones I would love for this to be shipped. Please let me know how I can help.

**Comment by tay1orjones** [comment]
@metonym I reached out to our team and @francinelucca has offered to walk through the tutorial as a second review/validation 🎉 

If there's no hiccups, I too would love to see this on the site.

File: src/pages/developing/svelte-tutorial/step-1.mdx [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
+    },
+  },
+};
+```
+
+Then, start the app again.
+
+```bash
+yarn dev
+```
```
Comment (francinelucca) [inline src/pages/developing/svelte-tutorial/step-1.mdx:176]: I haven't been able to succesfully run the dev server after applying these configurations:
image

Branch I'm working on: https://github.com/francinelucca/carbon-tutorial-svelte/tree/svelte-step-1

Comment (francinelucca) [reply src/pages/developing/svelte-tutorial/step-1.mdx:176]: CC @metonym

File: src/pages/developing/svelte-tutorial/step-1.mdx [hunk 2]
```diff
@@ -0,0 +1,486 @@
...
+Now let's import the icons. In the `TutorialHeader.svelte` script block, we need
+to import each individual icon we will use.
+
+```javascript path=src/lib/TutorialHeader.svelte
+import {
+  Notification20,
+  UserAvatar20,
+  AppSwitcher20,
+} from 'carbon-icons-svelte';
+```
```
Comment (francinelucca) [inline src/pages/developing/svelte-tutorial/step-1.mdx:310]: are we supposed to already have `carbon-icons-svelte` installed at this point? tutorial doesn't mention it 
image

File: src/pages/developing/svelte-tutorial/step-2.mdx [hunk 3]
```diff
@@ -0,0 +1,719 @@
...
+  content: '';
+  position: absolute;
+  z-index: -1;
+  top: 0;
+  bottom: 0;
+  right: -'spacing-05';
+  left: -'spacing-05';
+  background: 'ui-01';
+}
+```
```
Comment (francinelucca) [inline src/pages/developing/svelte-tutorial/step-2.mdx:403]: There is a visual difference in the grid background span, seems to be these styles which are included in https://svelte-step-3--carbon-tutorial-svelte.netlify.app/ but not in the tutorial:
image

Deployed Step 2:
image

Result from following tutorial:
image

File: src/pages/developing/svelte-tutorial/step-2.mdx [hunk 4]
```diff
@@ -0,0 +1,719 @@
...
+
+<InlineNotification>
+
+**Note:** Having issues pushing your changes?
+[Step 1](/developing/svelte-tutorial/step-1#git-commit-and-push) has
+troubleshooting notes that may help.
+
+</InlineNotification>
+
+### Pull request (PR)
```
Comment (francinelucca) [inline src/pages/developing/svelte-tutorial/step-2.mdx:706]: Opened PR but stuck in "👷 Deploy request for carbon-tutorial-svelte pending review." https://github.com/carbon-design-system/carbon-tutorial-svelte/pull/14

**Review by francinelucca — CHANGES_REQUESTED** [review]

File: src/pages/developing/svelte-tutorial/step-2.mdx [hunk 5]
```diff
@@ -0,0 +1,719 @@
...
+```html
+<Breadcrumb noTrailingSlash aria-label="Page navigation">
+```
+<!-- prettier-ignore-end -->
+
+Same goes for the `Tabs` opening tag:
+
+<!-- prettier-ignore-start -->
+```html
+<Tabs aria-label="Tab navigation">
```
Comment (francinelucca) [inline src/pages/developing/svelte-tutorial/step-2.mdx:277]: Tabs in completed step 2 tutorial differ from preview in the horizontal position, looks like it's missing these styles in the tutorial:
image
[Preview](https://svelte-step-3--carbon-tutorial-svelte.netlify.app/):
image
Tutorial:
image

File: src/pages/developing/svelte-tutorial/wrapping-up.mdx [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+  the Carbon Design System.
+tabs:
+  ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
+---
+
+<PageDescription>
+
+Thanks for completing our tutorial and helping us improve it along the way. If
+you want to show the world your new skills, you can apply for an IBM Digital
+Badge.
```
Comment (francinelucca) [inline src/pages/developing/svelte-tutorial/wrapping-up.mdx:14]: is there/will there be a badge process for this? if not maybe we'd want to replace the content in this page to something similar to https://carbondesignsystem.com/developing/react-tutorial/wrapping-up

**Review by francinelucca — CHANGES_REQUESTED** [review]
Outstanding tutorial PRs:
Step 2: https://github.com/carbon-design-system/carbon-tutorial-svelte/pull/14
Step 3: https://github.com/carbon-design-system/carbon-tutorial-svelte/pull/15
Step 4: https://github.com/carbon-design-system/carbon-tutorial-svelte/pull/16
Step 5: https://github.com/carbon-design-system/carbon-tutorial-svelte/pull/17
* Waiting on step 1 comments resolution for PR submission

Deployed tutorial: http://carbon-tutorial-svelte-francinelucca.mybluemix.net/

**Outstanding comments on steps 1 and 2

File: src/pages/developing/svelte-tutorial/wrapping-up.mdx (hunk 6 above)
Comment (tay1orjones) [reply src/pages/developing/svelte-tutorial/wrapping-up.mdx:14]: @francinelucca There will not - we've recently removed badging from all tutorials.

**Comment by tay1orjones** [comment]
Going to mark this as a draft for now

**Comment by alisonjoseph** [comment]
Closing for now, we can reopen in the future if needed.

**Pull request closed without merging**